test/nn/simplicial/test_dist2cycle.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for Dist2Cycke Model."""
+import numpy as np
+import numpy.linalg as npla
+import torch
+from toponetx.classes import SimplicialComplex as sc
+
+from topomodelx.nn.simplicial.dist2cycle import Dist2Cycle
+
+
+class TestDist2Cycle:
+    """Unit tests for the Dist2Cycle model class."""
+
+    def test_forward(self):
+        """Test the forward method of Dist2Cycle."""
+        edge_set = [[1, 2], [1, 3], [2, 5], [3, 5]]
+        face_set = [[2, 3, 4], [2, 4, 5]]
+
+        torch.manual_seed(42)
+        ex2_sc = sc(edge_set + face_set)
+        ld = ex2_sc.down_laplacian_matrix(rank=1).todense()
+        A = ex2_sc.adjacency_matrix(rank=1).todense()
```

Comment:
TopoModelX does not use the convention `A` to refer to the adjacency matrix.
For example:
https://github.com/pyt-team/TopoModelX/blob/486adf2299eb37aa22ec1fdf54351458074da253/topomodelx/nn/simplicial/hsn_layer.py#L114

Here, you should write `adjacency_1`.

Note: In Python, uppercase letters are reserved for class names (observe how they get colored in orange: orange means Python class). Thus, we should not use uppercase letters for variable names.